src/iocore/net/UnixNetVConnection.cc

```diff
@@ -663,7 +663,7 @@ UnixNetVConnection::net_write_io(NetHandler *nh)
       nh->write_ready_list.remove(this);
     }
 
-    int err, ret;
+    int err{0}, ret;
```

Comment:
Let's set `ret` to 0 as well.

Reply:
Thanks! Fixed at [afe6f76](https://github.com/apache/trafficserver/pull/12226/commits/afe6f767418cd7828d3aadcaef06fa63073ee1f9).